File: app/src/main/java/cg/lastfm/di/ArtistDetailsActivityModule.java
package cg.lastfm.di;

import cg.lastfm.ArtistDetailsActivity;
import cg.lastfm.MainActivity;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class ArtistDetailsActivityModule {
    @ContributesAndroidInjector
    abstract ArtistDetailsActivity contributeArtistDetailsActivity();
}

File: app/src/main/java/cg/lastfm/di/ArtistDetailsActivityModule.kt
package cg.lastfm.di

import cg.lastfm.ArtistDetailsActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class ArtistDetailsActivityModule {
    @ContributesAndroidInjector
    abstract fun contributeArtistDetailsActivity(): ArtistDetailsActivity
}
